docs/faqs/faqs.mdx

```diff
@@ -80,10 +80,10 @@ it wherever you want; on the public cloud, private cloud, or on-premise.
 
 ## How can I get the code?
 
-You can download the source code of your application or connect to your GitHub
-account and push a new Pull Request directly to a selected GitHub repository. There
+You can connect your project to your preferred git provider like [GitHub](/sync-with-github) or [Bitbucket](/sync-with-bitbucket).
+After connecting to a git repo you can push a new Pull Request directly to a selected git repository. There
 is also a code-view option, that enables you to view the code in the console before
-you download it or move it to GitHub.
+you download it or move it to your git provider.
```

Comment:
no option to download, please only leave the "push it to your git repository" (which is better than "move it to your git provider" IMO)